lib/mpl_toolkits/tests/test_axes_grid.py

```diff
@@ -44,7 +44,7 @@ def test_imagegrid_cbar_mode_edge():
         # "second" ones.  To achieve this, clear out the axes first.
         for ax in grid:
             ax.cax.cla()
-            cb = ax.cax.colorbar(ax.images[0])
+            ax.cax.colorbar(ax.images[0])
```

Comment:
Why did you drop the reference?

Reply:
It is never used, so when I was testing I'm guessing I got rid of it and didn't realize it was dropped at some point.